spec/spec_helper.rb

```diff
@@ -109,23 +109,30 @@ def webmachine_present?
   false
 end
 
-# Add way to clear subscribers between specs
-module ActiveSupport
-  module Notifications
-    class Fanout
-      def clear_subscribers
-        @subscribers.clear
-        @listeners_for.clear
-      end
-    end
-  end
+require 'appsignal'
+
+def spec_dir
+  File.dirname(__FILE__)
+end
+
+def tmp_dir
+  @tmp_dir ||= File.expand_path('tmp', spec_dir)
 end
 
+def fixtures_dir
+  @fixtures_dir ||= File.expand_path('support/fixtures', spec_dir)
+end
+
+def helpers_dir
+  @helpers_dir ||= File.expand_path('support/helpers', spec_dir)
+end
+
+Dir[File.join(helpers_dir, '*.rb')].each { |file| require file }
+
```

Comment:
This was probably readded by a rebase merge conflict?

These helpers are now located in the DirectoryHelper.

The helpers directory is already included on line 12-14